docs/guide/web-animation.md

```diff
@@ -17,11 +17,10 @@ The [figure below](#screenshot-of-a-web-animation-page-generated-by-webots) show
 
 ### How to Export a Web Animation
 
-Select the `File / Make HTML5 Animation...` menu item and choose the target `HTML` file.
+Select the `Share...` menu item and choose if you want to upload it to [Webots.cloud](https://webots.cloud) or to save it locally, then click the `Record and share animation` button to start the recording.
```

Comment:
Shouldn't be with lower case:
```suggestion
Select the `Share...` menu item and choose if you want to upload it to [webots.cloud](https://webots.cloud) or to save it locally, then click the `Record and share animation` button to start the recording.
```

I also checked and in the code and documentation of the preferences the URL is still `beta.webots.cloud` and not `webots.cloud`

Reply:
Yes, but it will be replaced by webots.cloud in the coming weeks, so I'm not sure it's worth changing it there and risk forgetting to change it again.